.github/workflows/playwright.yml

```
@@ -15,13 +15,19 @@ jobs:
     - name: Setup Playwright
       uses: ./.github/actions/playwright
     - name: Run Playwright tests
-      run: pnpm e2e
+      run: |
+        npx convex dev &        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)